freeipa.spec.in

```diff
@@ -503,6 +503,7 @@ Requires: python-custodia
 Requires: python-dns >= 1.11.1
 Requires: python-netifaces >= 0.10.4
 Requires: pyusb
+Requires: python-jinja2
```

Comment:
What about BuildRequires? pylint would be failing on the system where python-jinja2 is not installed.